Content.Server/GameTicking/Rules/ZombieRuleSystem.cs

```diff
@@ -69,16 +69,16 @@ private void OnRoundEndText(RoundEndTextAppendEvent ev)
             return;
 
         //this is just the general condition thing used for determining the win/lose text
-        var percent = Math.Round(GetInfectedPercentage(out var livingHumans), 2);
+        var percent = GetInfectedPercentage(out var livingHumans);
```

Comment:
Theres no need to round this one and the two below. You can just pick one or the other.

Reply:
Uh, that is precisely what I did:
I unrounded that one on line 72 and rounded the other two on line 79 and 81.
This way the value is rounded only after multiplication, so it is more precise in assessing the amount of zombies `zombie-round-end-amount-none`, `zombie-round-end-amount-low`, etc. and also will not return an exaggerate amount of decimal digits in-game.

Reply:
Whoops. Thats my bad for not being able to read diffs.